src/main/java/com/thealgorithms/strings/CheckVowels.java

```diff
@@ -14,19 +14,18 @@ public class CheckVowels {
     private static final Set<Character> VOWELS = new HashSet<>(Arrays.asList('a', 'e', 'i', 'o', 'u'));
 
     /**
-     * Check if a string is has vowels or not
+     * Check if a string has vowels or not
      *
      * @param input a string
      * @return {@code true} if given string has vowels, otherwise {@code false}
      */
     public static boolean hasVowels(String input) {
-        if (input == null) {
-            return false;
-        }
-        input = input.toLowerCase();
-        for (int i = 0; i < input.length(); i++) {
-            if (VOWELS.contains(input.charAt(i))) {
-                return true;
+        if (input != null) {
```

Comment:
I like the previous version more as it's easier to read because of simplified indentation